tests/ballerina-test/src/test/resources/test-src/io/clientsocketio.bal

```diff
@@ -0,0 +1,39 @@
+import ballerina.io;
+
+io:Socket socket;
+
+function openSocketConnection (string host, int port) {
+    socket = io:openSocket(host, port, {});
+}
+function openSocketConnectionWithProps (string host, int port, io:SocketProperties prop) (io:Socket) {
+    return io:openSocket(host, port, prop);
+}
+
+function isConnected () (boolean) {
+    return socket.isConnected();
+}
+
+function isClosed () (boolean) {
+    return socket.isClosed();
+}
+
+function closeSocket () {
+    socket.closeSocket();
+}
+
+function write (blob content) (int) {
+    io:ByteChannel channel = socket.channel;
+    return channel.writeBytes(content, 0);
+}
+
+function read (int size) (blob, int) {
+    io:ByteChannel channel = socket.channel;
+    blob readContent;
+    int readSize;
+    readContent, readSize = channel.readBytes(size);
+    return readContent, readSize;
+}
+
+function close(io:Socket socket) {
+    socket.closeSocket();
+}
```

Comment:
Shall we leave an empty space here ?

Reply:
46cd5053ef87419f9e68ead4cc40ddc161ccd086